{{cookiecutter.project_slug}}/Makefile

```diff
@@ -74,43 +74,43 @@ setup: ### Setup remote environment
 ##### STORAGE #####
 
 .PHONY: upload-code
-upload-code:  ### Upload code directory to Storage
+upload-code:  ### Upload code directory to your storage
```

Comment:
I'm not sure what's the terminology you stick to in other resources (tutorial, documentation), but bare phrase `Upload code directory to your storage` makes me struggle more than `... to Storage` because, from my point of view, "Storage" (with the capital letter) means a specific component of Neuro Platform, whereas "my storage" can mean anything -- my local directory, my dropbox, etc. So I would stick to more specific names like `... to Neuromation Storage`, or `... to your directory on Neuromation Storage`, something like that :smiley:

Reply:
👍